clang/include/clang/Driver/Options.td

```diff
@@ -6946,6 +6953,11 @@ class CLRemainingArgsJoined<string name> : Option<["/", "-"], name,
 // (We don't put any of these in cl_compile_Group as the options they alias are
 // already in the right group.)
 
+def _SLASH_Qfp_accuracy_EQ : CLJoined<"Qfp-accuracy=">,
+  Alias<ffp_accuracy_EQ>;
+def _SLASH_Qfp_accuracy_COL : CLJoined<"Qfp-accuracy:">,
+  Alias<ffp_accuracy_EQ>,HelpText<"Defines the accuracy for math library "
```

Comment:
For the Linux option the help string is "Specifies the required accuracy for floating-point operations and library calls". That's better and should be used here.